packages/ui-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@substrate/ui-components",
-  "version": "0.3.27",
+  "version": "0.3.31",
```

Comment:
Wait, why is the version bumped to 3.31 here and on npm?

I'd rather keep all versions in light-ui the same, and only bump via `yarn lerna version && yarn lerna publish` (exactly the same as on Nomidot).

Could you do a `yarn lerna version 0.3.32 && yearn learn publish` after merge?